config/routes.rb

```diff
@@ -113,26 +105,118 @@
 #
 # Routes for OpenApi::Rswag::Api::Engine:
 
+#                                                                 Prefix Verb   URI Pattern                                                                                                                              Controller#Action
```

Comment:
The routes are being duplicated here for some reason.

Reply:
I have regenerated. Looks like I messed up a rebase and didn't notice the duplicates